doc/content/integrations/storage/configuration.md

```diff
@@ -25,6 +25,16 @@ Initialize the configured database with:
 $ ttn-lw-stack storage-db init
 ```
 
+{{< note >}}
+If you are using TimescaleDB, the following configuration options are also available:
+
+```
+--timescaledb.chunk-time-interval
+--timescaledb.enable-retention-policy
+--timescaledb.retention-days
```

Comment:
This is only effective if the `--timescaledb.enable-retention-policy` flag is enabled as well.

This is the number of days for which received messages will be stored (based on the timestamp of the upstream message). The default value for this flag is `30`, which means that messages are stored for a month.